packages/react-native/React/CoreModules/RCTTiming.mm

```diff
@@ -128,6 +128,11 @@ - (void)setup
   _paused = YES;
   _timers = [NSMutableDictionary new];
   _inBackground = NO;
+  RCTExecuteOnMainQueue(^{
+    if (!self->_inBackground && [RCTSharedApplication() applicationState] == UIApplicationStateBackground) {
+      [self appDidMoveToBackground];
+    }
+  });
```

Comment:
Doesn't this need to be ifdefed?

Reply:
I had this change locally but I forgot to push it. 😅

Anyway, a54805f resolves this.